nrf_wifi/fw_if/umac_if/src/fmac_api_common.c

```diff
@@ -9,6 +9,7 @@
  * FMAC IF Layer of the Wi-Fi driver.
  */
 
+#include <string.h>
```

Comment:
Which change necessitated the inclusion of this header file?

Reply:
memset function requires this header.

Reply:
Ahh ok...these are OS agnostic files. So need to use `nrf_wifi_osal_mem_set` instead of `memset` directly.